### ISSUE TITLE
[BUG] No save button for zone selection with screen reader

### ISSUE THREAD

**Issue by havwig** [AtB-AS/mittatb-app#1062]
**Describe the bug**
The "Save" button is missing from the zone selection screen when using a screen reader.

**Application version**
Version number: 1.6

**To Reproduce**
1. Turn on screen reader
2. Go to zone selection for ticket purchase 
3. Try to change the selection

**Expected behavior**
Save button available (or that "back" saves, this should maybe be labled "cancel" instead to avoid confusion)

**Smartphone (please complete the following information):**
 - Device: Samsung Note 10
 - OS: Android 11
